runbook/README.md

```diff
@@ -0,0 +1,106 @@
+Once your instance is provisioned, be sure to put its IP address into the `HostName` section of the `~/.ssh/config` file on your local client so that you can connect to it from your
+client by simply typing `ssh staging` in your terminal window.
+
+If you do *not* have access to the `jh-covid` account, you should walk through the regular EC2 Launch Wizard flow and be sure to choose the
+*Amazon Linux 2 AMI (HVM), SSD Volume Type* (`ami-0e34e7b9ca0ace12d`, the 64-bit x86 version) AMI. Once the machine is up and running and you can
+SSH to it, you will need to run the following code to install the software you will need for the rest of the run:
+
+```
+sudo yum -y update
+sudo yum -y install awscli 
+sudo yum -y install git 
+sudo yum -y install docker.io 
+sudo yum -y install pbzip2 
+
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
+sudo yum -y install git-lfs
+git lfs install
+```
+
+## Connect to Github
+
+Once your staging server is provisioned and you can connect to it, you should `scp` the private key file that you use for connecting to
+Github to the `/home/ec2-user/.ssh` directory on the staging server (e.g., if the local file is named `~/.ssh/id_rsa`, then you should run
+`scp ~/.ssh/id_rsa staging:/home/ec2-user/.ssh` to do the copy. For convenience, you should create a `/home/ec2-user/.ssh/config` file on
+the staging server that has the following entry:
+
+```
+host github.com
+ HostName github.com
+ IdentityFile ~/.ssh/id_rsa
+ User git
+```
+
+This way, the `git clone`, `git pull`, etc. operations that you run on the staging server will use your SSH key without constantly prompting
+you to login. You should now be able to clone a COVID19 data repository into your home directory on the staging server to do work against.
+For this example, I'm going to use the `COVID19_Minimal` repo as my example, so I would run `git clone git@github.com:HopkinsIDD/COVID19_Minimal.git`
+to get it onto the staging server. By convention, I usually do runs (for both Planning and Inference jobs) with the `COVIDScenarioPipeline` repository
+nested inside of the data repository, so I would then do `cd COVID19_Minimal; git clone git@github.com:HopkinsIDD/COVIDScenarioPipeline.git` to
+clone the modeling code itself into a child directory of the data repository.
+
+## Getting and Launching the Docker Container
+
+The previous section is only for getting a minimal set of dependencies setup on your staging server. To do an actual run, you will
+need to download the Docker container that contains the more extensive set of dependencies we need for running the code in the
+`COVIDScenarioPipeline` repository. For *Inference* jobs, please run `sudo docker pull hopkinsidd/covidscenariopipeline:latest-dev`;
```

Comment:
i always use the dev container for both planning and inference

Reply:
That would certainly simplify things. 🤔